modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/StructuredContentFolderResourceImpl.java

```diff
@@ -402,6 +448,14 @@ private Page<StructuredContentFolder> _getFoldersPage(
 					GetterUtil.getLong(document.get(Field.ENTRY_CLASS_PK)))));
 	}
 
+	private JournalFolder _getStructuredContentFolderByExternalReferenceCode(
```

Comment:
To be honest, I don't know if makes sense to have a private method with only one instruction

Reply:
Good catch. Better to inline this where it is used